eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/RSMergeWriter.java

```diff
@@ -192,6 +193,18 @@ public void writeRSMerge(SubmittedVariantOperationEntity currentOperation)
                 getMergeDestinationAndMergees(mergeCandidates);
         ClusteredVariantEntity mergeDestination = mergeDestinationAndMergees.getLeft();
 
+        // check if any variant with same hash as mergeDestination already exist in DB
```

Comment:
This section is only relevant if the list of `mergees` has more than 2 elements. Can we test this before going through this ?

Reply:
Does it make more sense to actually just extract [this removal](https://github.com/EBIvariation/eva-accession/blob/2f2913383590020f6d5a1fd8a60caca5c75cf975/eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/RSMergeWriter.java#L315) into a method that's called before the for-loop, and modify it so it checks for all mergee accessions instead of just one?

I'm sure this implementation works as well, but I'm wary of essentially redo-ing merge detection and prioritisation logic rather than just trusting the candidates we have.